docs/resources/aide_conf.md.erb

```diff
@@ -56,16 +60,17 @@ The following examples show how to use this InSpec audit resource. For a full li
       its('rules') { should include ['r', 'sha512'] }
     end
 
+### The usage of all\_have\_rule will return whether or not all selection lines in audit.conf contain a particular rule:
+
+    describe aide_conf.all_have_rule('sha512') do
+      it { should eq true }
+    end
 <br>
 
 ## Matchers
 
-This InSpec audit resource has the following matchers. For a full list of available matchers please visit our [matchers page](https://www.inspec.io/docs/reference/matchers/).
+This InSpec audit resource uses the matchers `eq` and `include`. 
```

Comment:
I think this line should be removed. Our examples might use these matchers, but all the standard matchers (of which `eq` and `include` are a member) can apply to this resource. Referring to the full matcher list should be sufficient.

Reply:
OK. Can we remove it everywhere?